app/Http/Controllers/AuthController.php

```diff
@@ -35,6 +37,7 @@ public function login(Request $request)
         return response()->json([
             'access_token' => $user->createToken('api_token')->plainTextToken,
             'token_type' => 'Bearer',
+            'user' => $user
```

Comment:
Remove user returned in JSON after successful registration. Only thing needed is the token

Reply:
ok i did it